### PR TITLE
Resolving python compatibility issue with decode.

### PR DESCRIPTION
**Pull request by kwoodson**

### CODE REVIEW

File: roles/lib_openshift/library/oc_adm_ca_server_cert.py [hunk 1]
```diff
@@ -1508,7 +1508,7 @@ def exists(self):
         x509output, _ = proc.communicate()
         if proc.returncode == 0:
             regex = re.compile(r"^\s*X509v3 Subject Alternative Name:\s*?\n\s*(.*)\s*\n", re.MULTILINE)
-            match = regex.search(x509output)  # E501
+            match = regex.search(x509output.decode())  # E501
```
Comment (ashcrow) [inline roles/lib_openshift/library/oc_adm_ca_server_cert.py]: This should work since it will be ``bytes`` on 3 and ``str`` on 2.

**Comment by ashcrow** [comment]
aos-ci-test

**Comment by ashcrow** [comment]
[merge]

**Comment by ashcrow** [comment]
I'm guessing this is a flake. I've seen it with another PR already.
```
TASK [openshift_node : Start and enable node] **********************************
FAILED - RETRYING: TASK: openshift_node : Start and enable node (1 retries left).
fatal: [10.8.170.80]: FAILED! => {"attempts": 1, "changed": false, "failed": true, "msg": "Unable to start service atomic-openshift-node: Job for atomic-openshift-node.service failed because the control process exited with error code. See \"systemctl status atomic-openshift-node.service\" and \"journalctl -xe\" for details.\n"}
```

**Comment by kwoodson** [comment]
flake https://github.com/openshift/origin/issues/13271

**Comment by kwoodson** [comment]
[merge]

**Comment by kwoodson** [comment]
Flake https://github.com/openshift/origin/issues/13271

**Comment by kwoodson** [comment]
[merge]

**Comment by kwoodson** [comment]
It appears that there was something went completely wrong during the build process.  This is becoming the norm.

This is probably missing or couldn't build pyopenssl.

```
    build/temp.linux-x86_64-2.7/_openssl.c:12:24: fatal error: pyconfig.h: No such file or directory
     #  include <pyconfig.h>
                            ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
```

**Comment by kwoodson** [comment]
aos-ci-test

**Comment by kwoodson** [comment]
Need tox test results to pass.

**Comment by ashcrow** [comment]
```
    build/temp.linux-x86_64-2.7/_openssl.c:12:24: fatal error: pyconfig.h: No such file or directory
     #  include <pyconfig.h>
```

That is ... insane.

**Comment by kwoodson** [comment]
Maybe it was python-devel.

**Comment by kwoodson** [comment]
[merge]

**Pull request merged**